recipes/md/trajectoryanalysis.py

```diff
@@ -165,17 +165,129 @@ def postrun(self):
             f.write(f"{D}\n")
 
 
+def viscosity_double_exponential(x, A, lam, tau1, tau2):
+    return A * (lam * (1 - np.exp(-x / tau1)) + (1 - lam) * (1 - np.exp(-x / tau2)))
+
+
+class AMSViscosityFromBinLogResults(AMSAnalysisResults):
```

Comment:
This name does not really speak to me... but I see the `BinLog` is a documented thing so fair enough